js/index.js

```diff
@@ -1,7 +1,49 @@
 // Iteration 1: Names and Input
 
-
 // Iteration 2: Conditionals
 
+// Iteration 3: Loops
+
+console.log("I'm ready");
+let hacker1 = "Ciwan";
+console.log("The driver's name is " + hacker1);
+let hacker2 = "Vasiliki";
+console.log("The navigator's name is " + hacker2);
+
+if (hacker1.length > hacker2.length) {
+  console.log(
+    "The driver has the longest name, it has" + hacker1.length + " charechters"
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    "It seems that the navigator has longest name, it has " +
+      hacker2.length +
+      " characters."
+  );
+} else {
+  console.log(
+    "Wow, you have both equally long names, " + hacker1.length + " characters!"
+  );
+}
+
```

Comment:
All good in the hood guys!
Only comment about this iteration is the same as the comment above regarding the variables in a string :D